influxql/functions_test.go

```diff
@@ -384,3 +386,98 @@ func TestHoltWinters_MaxTime(t *testing.T) {
 		}
 	}
 }
+
+// TestSample_IsRandom attempts to verify that the subset of data that is returned
+// by sample is actually random.
+func TestSample_IsRandom(t *testing.T) {
```

Comment:
Can you explain how this function works? I'm a bit confused about how it verifies the randomness of the sample.